CMakeLists.txt

```diff
@@ -607,13 +628,39 @@ endif()
 install(
   TARGETS surelog-bin
   RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
+
+# Libraries
+install(
+  TARGETS surelog
+  EXPORT Surelog
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}/surelog
+  PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/Surelog)
+
```

Comment:
For some of the following libraries, the question is if we even need to install them as they are static libraries. I guess it might depend on system-specific link rules if they are needed 'in production'. Something to worry about in follow-up cleanups.